unified/install.sh

```diff
@@ -88,6 +86,31 @@ if [ -z "$prefix" ]; then
     fi
 fi
 
+if [ -z "$python3" ]; then
+    if $nonroot; then
+        python3=$prefix/python3/bin/python3
+    else
+        python3=$prefix/python3/bin/python3
+    fi
+fi
+
+if [ -f "/etc/os-release" ]; then
+    . /etc/os-release
+elif [ -f "/etc/arch-release" ]; then
+    export ID=arch
+else
+    echo "/etc/os-release missing."
+    exit 1
+fi
+
+if [ -z "$sysconfdir" ]; then
+    if [ "$ID" = "ubuntu" ] || [ "$ID" = "debian" ]; then
+        sysconfdir=/etc/default
+    else
+        sysconfdir=/etc/sysconfig
+    fi
+fi
+
```

Comment:
Current script designed to use /etc/sysconfig even on Debian variants when --sysconfdir is not specified, on offline & nonroot install.
We can change this, but we should apply this on install.sh not just unified/install.sh, just like --python3 option.
And also install.sh on scylla-tools & scylla-jmx & scylla-python3, too.

Reply:
Let's continue using /etc/sysconfig for root install of debian-like distro.
I will change this back.